packages/projection/src/operators/Mappers/withHandleMetadata.ts

```diff
@@ -0,0 +1,101 @@
+import { Asset, Cardano, Handle } from '@cardano-sdk/core';
+import { CIP67Assets, WithCIP67 } from './withCIP67';
+import { FilterByPolicyIds } from './types';
+import { Logger } from 'ts-log';
+import { ProducedUtxo } from './withUtxo';
+import { ProjectedNftMetadata, WithNftMetadata } from './withNftMetadata';
+import { ProjectionOperator } from '../../types';
+import { assetNameToUTF8Handle } from './util';
+import { isNotNil } from '@cardano-sdk/util';
+import { map } from 'rxjs';
+
+interface PersonalizedProperties {
+  profilePicImage?: Asset.Uri;
+  backgroundImage?: Asset.Uri;
+  /**
+   * Only present for cip68/personalized handles
```

Comment:
This could be hoisted to describe the whole interface

Reply:
Done in 446ce7b24

Yes, cf1dfc9b8a3560a8731ad953f3704765c33f7865 could potentially resolve the discrepancy